addons/im_odoo_support/static/src/js/im_odoo_support.js

```diff
@@ -44,6 +44,9 @@
             }
         },
         start_support: function(){
+            // The free livechat service doesn't exist anymore
+            window.open('https://www.odoo.com/help', '_blank');
+            return;
```

Comment:
IMHO, the complete module could be removed since we don't provide this service anymore. Since it is 8.0, I preferred keeping the diff short instead of butchering the module.